README.MD

```diff
@@ -165,8 +168,17 @@ You can change the report name to a name you want
 
 You can customise Page Footer if required. You just need to provide a html string like `<div><p>A custom footer in html</p></div>`
 
+### `plainDescription`
+- **Type:** `boolean`
+- **Default:** `false`
+- **Mandatory:** No
+
+Feature descriptions are nicely formatted by the template. If your test features provide already a custom formatting
```

Comment:
Can you explain a bit more what you mean with 

> If your test features provide already a custom formatting

so it will be more clear for people?

Reply:
@wswebcreation rewrote the section completely. Really hope now it's clearer than before.